.github/actions/test-dotnet/Set-XUnitMaxParallelThreads.ps1

```diff
@@ -1,19 +1,19 @@
 ﻿param (
-    [Parameter(Mandatory=$true)]
+    [Parameter(Mandatory = $true)]
     $MaxParallelThreads
 )
 
 Write-Output "Setting maxParallelThreads in xunit.runner.json files to $MaxParallelThreads."
 
 $parameters = @{
-        # -Filter is the fastest way to find files because its value is passed directly to the file system API.
-        Filter = "xunit.runner.json"
-        Recurse = $true
-        # -FollowSymLink is needed because without it errors will be thrown of type:
-        # "Get-ChildItem: The system cannot find the path specified."
-        FollowSymlink = $true
-        # Ignore warnings that will be printed due to multiple symlinks pointing to the same physical file.
-        WarningAction = "Ignore"
+    # -Filter is the fastest way to find files because its value is passed directly to the file system API.
+    Filter        = "xunit.runner.json"
+    Recurse       = $true
```

Comment:
Such tab-arranged assignments are not something we do anywhere, so let's not start it in just this script.

Reply:
> I'd prefer not to do this tab stop-alignment here either.

Agreed, for reference, I replied on another PR: https://github.com/Lombiq/Utility-Scripts/pull/28#discussion_r1051637023